img/p/pro.btcturk.com.svg

```diff
@@ -0,0 +1,8 @@
+</g>
+</svg>
```

Comment:
This SVG file just contains an embedded PNG file.
I recommend you use this one instead:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 35.97 47.98"><linearGradient id="a" x1="3" x2="40" y1="25.61" y2="25.61" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#003aff"/><stop offset=".25" stop-color="#0067ff"/><stop offset=".8" stop-color="#02d6ff"/><stop offset="1" stop-color="#02ffff"/></linearGradient><path fill="url(#a)" d="M23.6 41.97c-2.75 1.98-4.4 4.44-3.64 5.53.76 1.08 3.61.35 6.37-1.63 2.76-1.98 4.4-4.45 3.64-5.53-.22-.36-.66-.48-1.27-.48h-.08c-1.26 0-3.15.76-5.02 2.1M8.4 37.75C3.05 39.98-.66 43.31.1 45.2c.76 1.9 5.73 1.6 11.09-.6 5.35-2.24 9.06-5.57 8.3-7.45-.35-.83-1.52-1.25-3.2-1.25h-.15c-2.08 0-4.87.64-7.74 1.85m24.52-2.23c-.95 1.82-1.24 3.55-.67 3.83.58.32 1.81-.93 2.76-2.74.95-1.83 1.24-3.56.67-3.84-.06-.03-.13-.03-.2-.03h-.01c-.64 0-1.7 1.17-2.54 2.78m-7.73-5.05c-1.84 1.05-2.8 2.84-2.16 4 .67 1.14 2.66 1.18 4.5.12 1.84-1.05 2.79-2.85 2.15-4-.34-.6-1.04-.9-1.9-.9-.79 0-1.7.26-2.6.78m6.78-2.43c-.53.92-.5 1.94.1 2.3.57.35 1.5-.16 2-1.09.54-.93.5-1.95-.09-2.3a.89.89 0 0 0-.41-.1c-.51 0-1.14.48-1.59 1.19m-6.81-4c0 1.12 1.01 2.01 2.25 2.01 1.23 0 2.28-.9 2.28-2.01 0-1.12-1.01-2.02-2.28-2.02-1.24 0-2.25.93-2.25 2.02m-15.1 0c0 2.5 2.78 4.5 6.23 4.5 3.42.04 6.21-1.98 6.21-4.5 0-2.5-2.79-4.51-6.24-4.51-3.42 0-6.2 2.01-6.2 4.5m22.07-6.36c-.57.36-.6 1.38-.1 2.3.54.93 1.43 1.41 2 1.1.57-.36.6-1.38.09-2.3-.41-.74-1.05-1.19-1.58-1.19a.36.36 0 0 0-.08 0 .7.7 0 0 0-.33.1m-9-4.13c-.67 1.15.3 2.94 2.14 4 1.84 1.05 3.84 1.02 4.5-.13.67-1.15-.31-2.94-2.15-4a5.34 5.34 0 0 0-2.6-.76c-.86 0-1.58.28-1.9.89m9.12-4.9c-.57.33-.28 2.02.67 3.84.95 1.83 2.19 3.04 2.76 2.75.57-.32.28-2.01-.67-3.83-.85-1.63-1.93-2.79-2.57-2.79a.4.4 0 0 0-.19.04M.1 2.8C-.66 4.68 3.05 8 8.4 10.25c5.35 2.24 10.3 2.5 11.09.6.76-1.88-2.95-5.2-8.3-7.45-2.98-1.21-5.8-1.85-7.9-1.85-1.67 0-2.84.41-3.2 1.25M19.94.5c-.76 1.08.89 3.54 3.64 5.53 2.76 1.98 5.61 2.71 6.37 1.63.76-1.09-.89-3.55-3.64-5.54C24.4.78 22.46.02 21.2.02c-.6 0-1.05.16-1.27.47"/></svg>
```